examples/lehrfempp/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-lehrfempp)
+
+# DOCUMENTATION_START {
+hunter_add_package(lehrfempp)
+find_package(lehrfempp CONFIG REQUIRED)
+
+add_executable(boo boo.cpp)
+target_link_libraries(boo PUBLIC lf::base)
```

Comment:
It's up to you but in my opinion it will look better if namespace in target will match package name:
```cmake
hunter_add_package(lehrfempp)
find_package(lehrfempp CONFIG REQUIRED)
target_link_libraries(boo PUBLIC lehrfempp::base)
```
Same in C++:
```cpp
#include <lehrfempp/base.h>

int main() {
  lehrfempp::foo();
}
```